webapp/views/App/views/Data/DataImport/DataImportCsvView.js

```diff
@@ -67,8 +89,23 @@ export const DataImportCsvView = () => {
         </FormItem>
       )}
 
+      <FormItem label={i18n.t('dataImportView.importType.label')}>
+        <ButtonGroup
+          selectedItemKey={dataImportType}
+          onChange={onDataImportTypeChange}
+          items={Object.values(importTypes).map((importType) => ({
+            key: importType,
+            label: i18n.t(`dataImportView.importType.${importType}`),
+          }))}
+        />
+      </FormItem>
+
       <FormItem label={i18n.t('dataImportView.importIntoEntity')}>
-        <EntitySelectorTree nodeDefUuidActive={selectedEntityDefUuid} onSelect={onEntitySelect} />
+        <EntitySelectorTree
+          nodeDefUuidActive={selectedEntityDefUuid}
+          onSelect={onEntitySelect}
+          isDisabled={() => insertNewRecords}
```

Comment:
is insertNewRecords a function?

Reply:
insertNewRecords is a boolean variable, defined few lines above and depending on the dataImportType parameter, but I will change it